frontend/src/api/network/notebooks.ts

```diff
@@ -87,8 +88,9 @@ const assembleNotebook = (data: StartNotebookData, username: string): NotebookKi
     tolerationSettings,
   } = data;
   const notebookId = overrideNotebookId || translateDisplayNameForK8s(notebookName);
+  const { notebookNamespace, dashboardNamespace } = useNamespaces();
```

Comment:
```suggestion
  const { dashboardNamespace } = useNamespaces();
```
`notebookNamespace` does not exist for DS Projects (aka DSG) -- but I believe you're also not using it.

Reply:
Yes, I am only using dashboardNamespace, as that is where the image trigger annotation imagestream is in.